modules/articles/tests/client/articles.client.controller.tests.js

```diff
@@ -1,14 +1,14 @@
-'use strict';
-
 (function () {
+  'use strict';
+
   // Articles Controller Spec
   describe('Articles Controller Tests', function () {
     // Initialize global variables
     var ArticlesController,
       scope,
+      rootScope,
```

Comment:
@codydaig I am honestly not the best with tests.  I didn't write them to begin with and was just making them "work" with the new style changes.  What would you say to merging this PR and fixing the styles on the tests at a later time?  or do you want to submit a PR to update the tests to this PR?  I just don't know enough about jasmine to do this solo